components/moderator/OrgCard/index.tsx

```diff
@@ -5,14 +5,55 @@ import styles from './OrgCard.module.css';
 
 type CardProps = {
   org: Organization;
+  orgIndex: number;
+  pageIndex: number;
 };
 
-const OrgCard: React.FunctionComponent<CardProps> = ({ org }) => {
+const OrgCard: React.FunctionComponent<CardProps> = ({
+  org,
+  orgIndex,
+  pageIndex,
+}) => {
+  if (orgIndex !== pageIndex) {
+    return (
+      <Card
+        className={styles.root}
+        classes={{
+          root: styles.cardRoot,
+        }}
+        raised={false}
+      >
+        <CardActionArea>
+          <div>
+            <CardContent className={styles.content}>
+              <Typography className={styles.date} component="h6" variant="h6">
+                {org.updatedAt.toDateString()}
+              </Typography>
+              {org.name && (
+                <Typography component="h5" variant="h5">
+                  {org.name}
+                </Typography>
+              )}
+            </CardContent>
+          </div>
+        </CardActionArea>
+      </Card>
+    );
+  }
   return (
-    <Card className={styles.root}>
+    <Card
+      className={styles.rootChosen}
+      classes={{
+        root: styles.cardRoot,
+      }}
+      raised={false}
+    >
       <CardActionArea>
-        <div className={styles.cardMain}>
-          <CardContent className={styles.content}>
+        <div>
+          <CardContent className={styles.contentChosen}>
+            <Typography className={styles.date} component="h6" variant="h6">
+              {org.updatedAt.toDateString()}
+            </Typography>
             {org.name && (
               <Typography component="h5" variant="h5">
```

Comment:
Can the org name size be changed to "body bold" size rather than h5? (If this is not the right spot sorry!)